### DISCUSSION TITLE
Number of Invalid plots keeps on increasing

### DISCUSSION THREAD

**Discussion by sauravkumal** [Chia-Network/chia-blockchain#7887] [Support]
Why is it that the number of invalid plots keep on increasing. I delete them and after few days there are more of them.
![Screenshot from 2021-07-31 21-40-57](https://user-images.githubusercontent.com/46340985/127745550-011e275a-53dc-4c88-aa4f-d48fea33fd97.png)

**Comment by keliew** [comment]
Check logs.
